src/bindings/jna/libelektra/src/main/java/org/libelektra/Elektra.java

```diff
@@ -390,6 +392,10 @@ int elektraGOptsContractFromStrings (Pointer contractKeySet, long argsSize, Stri
 
 	@Nullable Pointer ksLookupByName (Pointer ks, String name, int options);
 
+	int ksSearchInternal (Pointer keySet, Pointer key);
```

Comment:
This is not public API and as such it should not be used.

But probably it makes sense to provide `ksSearch`. The return type needs to be fixed though (elektraCursor).

Reply:
tracked via #4026 and implemented in this PR.